api/python/cellxgene_census/tests/test_open.py

```diff
@@ -13,11 +13,21 @@
 
 import cellxgene_census
 from cellxgene_census._open import DEFAULT_TILEDB_CONFIGURATION
-from cellxgene_census._release_directory import CELL_CENSUS_RELEASE_DIRECTORY_URL
+from cellxgene_census._release_directory import CELL_CENSUS_MIRRORS_DIRECTORY_URL, CELL_CENSUS_RELEASE_DIRECTORY_URL
+
+# Temporary mock for the mirrors.json route
+# Remove once the mirror.json route is released.
+mirrors_json = {
+    "default": "private",
+    "private": {"provider": "S3", "base_uri": "s3://cellxgene-data-public/", "region": "us-west-2"},
+}
 
 
 @pytest.mark.live_corpus
-def test_open_soma_stable() -> None:
+def test_open_soma_stable(requests_mock: rm.Mocker) -> None:
+    requests_mock.real_http = True
+    requests_mock.get(CELL_CENSUS_MIRRORS_DIRECTORY_URL, json=mirrors_json)
+
```

Comment:
nit: can we clean up the detritus in lines 42-47 (it appears identical to that on lines 36-40, albeit commented out)

Reply:
Removed.